cluster/gke/util.sh

```diff
@@ -166,6 +168,12 @@ function kube-up() {
     create_args+=("--additional-zones=${ADDITIONAL_ZONES}")
   fi
 
+  if [[ ! -z "${CLUSTER_IP_RANGE:-}" ]]; then
+    create_args+=("--cluster-ipv4-cidr=${CLUSTER_IP_RANGE}")
+  fi
+
+  create_args+=( ${GKE_CREATE_FLAGS:-} )
```

Comment:
It's intentional - it implicitly splits across spaces and dumps it in the array, e.g.:

```
$ LIST="foo bar blat"
$ ARR=( ${LIST} )
$ echo ${ARR[2]}
blat
```